src/util/config.rs

```diff
@@ -715,6 +715,8 @@ mod check_data_dir {
             static ref GETMNTENT_LOCK: Mutex<()> = Mutex::new(());
         }
 
+        let op = "data-dir.fsinfo.get";
```

Comment:
can we move the op to the error directly?

Reply:
seem only one error use it

Reply:
At the moment that may be the case, but a new error may be added. Also, this makes for consistency with other functions.